pkg/runner/runner.go

```diff
@@ -255,3 +261,23 @@ func Run(cfg *Config) error {
 	}
 	return err
 }
+
+func registerTelemetryMetrics(client *pgclient.Client) {
+	t, err := telemetry.NewEngine(client.Connection, PromscaleID, client.Queryable())
+	if err != nil {
+		log.Debug("msg", "err creating telemetry engine", "err", err.Error())
+		return
+	}
+	t.Start()
+	defer t.Stop()
```

Comment:
This doesn't seem right. Why are we stopping the engine upon telemetry registration?

Reply:
I'd recommend returning the engine instance from this function and invoking `Stop` in our shutdown/cleanup stage...

Reply:
Oh yes, you are right